aspnet/mvc/overview/older-versions-1/contact-manager/iteration-1-create-the-application-cs.md

```diff
@@ -436,7 +436,7 @@ The content of the Delete view is contained in Listing 9. This view contains a f
 
 ## Changing the Name of the Default Controller
 
-It might bother you that the name of our controller class for working with contacts is named the HomeController class. Shouldn t the controller be named ContactController?
+It might bother you that the name of our controller class for working with contacts is named the HomeController class. shouldn't the controller be named ContactController?
```

Comment:
```suggestion
It might bother you that the name of our controller class for working with contacts is named the HomeController class. Shouldn't the controller be named ContactController?
```